ckanext/dcat/urls.py

```diff
@@ -0,0 +1,23 @@
+import six
+from ckan.plugins import toolkit as tk
+from ckantoolkit import url_for as core_url_for
+
+
+def url_for(*args, **kwargs):
+
+    if not tk.check_ckan_version(min_version='2.9'):
+
+        external = kwargs.pop('_external', False)
+        if external is not None and 'qualified' not in kwargs:
+            kwargs['qualified'] = external
+
+        if len(args) and args[0] == 'dcat.read_dataset':
```

Comment:
Might be worth extracting `args[0]` to a local variable.

Reply:
This is already merged @ThrawnCA. Maybe you could add a new PR [here](https://github.com/ckan/ckanext-dcat/blob/master/ckanext/dcat/urls.py)